apps/api-gateway/src/chat/chat.controller.ts

```diff
@@ -1,15 +1,31 @@
-import { Controller, Post, Body } from '@nestjs/common';
+import { Controller, Post, Body, Get, Param, Query } from '@nestjs/common';
 import { ApiTags } from '@nestjs/swagger';
 import { ChatService } from './chat.service';
-import { CreateChatRoomDto } from './dto/req.dto';
+import { CreateChatRoomDto, GetChatListDto, GetChatRoomDto, GetChatRoomsDto } from './dto/req.dto';
 
 @Controller('chat')
 @ApiTags('Chat API')
 export class ChatController {
   public constructor(private readonly chatService: ChatService) {}
 
-  @Post('create-chat-room')
+  @Post('chat-rooms')
   public createChatRoom(@Body() createChatRoomDto: CreateChatRoomDto) {
     return this.chatService.createChatRoom(createChatRoomDto.name);
   }
+
+  @Get('chat-rooms')
+  public getChatRooms(@Body() { id }: GetChatRoomsDto) {
+    //! id값은 userId로 인증/인가 절차 구현시 삭제 예정
```

Comment:
요런건 todo 주석으로 표현되면 좋을 것 같습니다~
```suggestion
    // TODO: id값은 userId로 인증/인가 절차 구현시 삭제 예정
```